python/orca/tutorial/NCF/run.md

```diff
@@ -91,11 +168,29 @@ ${SPARK_HOME}/bin/spark-submit \
    --driver-memory 2g \
    --archives environment.tar.gz#environment \
    --properties-file ${BIGDL_HOME}/conf/spark-bigdl.conf \
-   --conf spark.pyspark.driver.python=/home/anaconda3/envs/NCF-yarn/bin/python \
+   --conf spark.pyspark.driver.python=python \
    --conf spark.pyspark.python=environment/bin/python \
-   --py-files ${BIGDL_HOME}/python/bigdl-spark_${SPARK_VERSION}-${BIGDL_VERSION}-python-api.zip,pytorch_model.py \
+   --py-files ${BIGDL_HOME}/python/bigdl-spark_${SPARK_VERSION}-${BIGDL_VERSION}-python-api.zip,process_spark_dataframe.py,pytorch_model.py,utils.py \
```

Comment:
same as above, does client mode need this two files?

Reply:
no, my bad:cry: